src/UIKit/UIListSeparatorConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+//
+// UIListSeparatorConfiguration.cs
+//
+// Authors:
+//	Alex Soto <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+#if IOS || WATCH
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+
+namespace UIKit {
+	public partial class UIListSeparatorConfiguration {
+
+		public static NSDirectionalEdgeInsets AutomaticInsets { get; private set; }
+
+		static UIListSeparatorConfiguration ()
```

Comment:
hmm... that makes the property "un-link'able" since the .cctor will always be preserved if the type is used

Maybe just lazy fetch (and cache) it on `get` ?

Reply:
Who can think of the linker!!! Thanks good catch!